Migrate SQL docs (#521)
redpanda-data/cloud-docs: modules/reference/pages/sql/sql-functions/other-functions/col-description.adoc

= col_description()
:description: The link:https://www.postgresql.org/docs/current/functions-info.html#FUNCTIONS-INFO-COMMENT[col_description()^] is a comment information function that 
:page-topic-type: reference

The link:https://www.postgresql.org/docs/current/functions-info.html#FUNCTIONS-INFO-COMMENT[col_description()^] is a comment information function that retrieves the comment associated with a specified table column.

== Syntax

The syntax for the `col_description()` function is as follows:

....
col_description (table_oid, column_number) → NULL
....

== Parameters

The following parameters are required to execute this function:

> * link:https://www.postgresql.org/docs/current/datatype-oid.html[table_oid^]: specifies the object identifier (OID) of the table containing the column for which you want to retrieve the comment

Reviewer: ```suggestion
* link:https://www.postgresql.org/docs/current/datatype-oid.html[table_oid^]: specifies the object identifier (OID) of the table containing the column from which to retrieve the comment
```
